lib/fabric/app/models/fabric/invoice.rb

```diff
@@ -83,13 +80,12 @@ def sync_with(invoice)
       self.amount_due = invoice.amount_due
       self.amount_paid = invoice.amount_paid
       self.amount_remaining = invoice.amount_remaining
-      self.application_fee = invoice.application_fee
+      self.application_fee_amount = invoice.application_fee_amount
```

Comment:
Did you run this in the console on an invoice and make sure this method is always defined? There are some fields that only exist in certain circumstances, and this one is marked "Connect only", which makes me worry that it might not be defined in some cases. I tried running it with our current code and it fails, but I don't have fabric set to this branch, so I don't know if it works in the stripe version you're upgrading to.

Reply:
You'd probably have to be sending the proper Stripe API version with your request if I had to guess.

Reply:
OK, yeah, I tested it with the new API version and it returns nil instead of a NoMethodError. So we're good.